subprojects/docs/src/docs/userguide/javaPlugin.adoc

```diff
@@ -647,14 +647,17 @@ For more details and examples please see the api:org.gradle.api.tasks.testing.Te
 Some examples of using the command line option:
 
 * `gradle test --tests org.gradle.SomeTest.someSpecificFeature`
```

Comment:
❌ We are missing some explanations what the differences are. Maybe we should also reduce the number of examples to a few recommended ones to make it easier to understand. Right now I'd just be very confused when reading this.